### PR TITLE
Account recovery

### PR DESCRIPTION
**Pull request by vgrichina**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -93,6 +59,48 @@ router.post('/account', async ctx => {
     ctx.body = response;
 });
 
+const password = require('secure-random-password');
+const models = require('./models');
+const FROM_PHONE = '+14086179592';
```
Comment (janedegtiareva) [inline app.js]: what is this phone number? comment please

File: app.js [hunk 2]
```diff
@@ -93,6 +59,48 @@ router.post('/account', async ctx => {
...
+router.post('/account/:phoneNumber/:accountId/requestCode', async ctx => {
+    const accountId = ctx.params.accountId;
+    const phoneNumber = ctx.params.phoneNumber;
+
+    const securityCode = password.randomPassword({ length: 6, characters: password.digits });
```
Comment (janedegtiareva) [inline app.js]: define 6 as a const? (also is 6 too few chars?)

File: app.js [hunk 3]
```diff
@@ -93,6 +59,48 @@ router.post('/account', async ctx => {
...
+    const [account] = await models.Account.findOrCreate({ where: { accountId, phoneNumber } });
+    await account.update({ securityCode });
+
+    const accountSid = process.env.TWILIO_ACCOUNT_SID;
```
Comment (janedegtiareva) [inline app.js]: do we need changes in readme how to set these?

File: app.js (hunk 1 above)
Comment (vgrichina) [reply app.js]: it is a a phone number from which codes are sent, thought this is more more or less obvious. Added env variable as well, going to document that in README

File: app.js (hunk 2 above)
Comment (vgrichina) [reply app.js]: Done

File: app.js (hunk 1 above)
Comment (vgrichina) [reply app.js]: README done

File: app.js (hunk 3 above)
Comment (vgrichina) [reply app.js]: done

**Pull request merged**